index.bs

```diff
@@ -4129,10 +4129,10 @@ An [=interface=] can be declared to be asynchronously iterable by using an
 
 <pre highlight="webidl" class="syntax">
     interface interface_identifier {
-      async iterable&lt;value_type&gt;;
-      async iterable&lt;value_type&gt;(/* arguments... */);
-      async iterable&lt;key_type, value_type&gt;;
-      async iterable&lt;key_type, value_type&gt;(/* arguments... */);
+      async_iterable&lt;value_type&gt;;
+      async_iterable&lt;value_type&gt;(/* arguments... */);
+      async_iterable&lt;key_type, value_type&gt;;
+      async_iterable&lt;key_type, value_type&gt;(/* arguments... */);
```

Comment:
This is not IDL type but the async iterable declaration, which should not change. Multiple people are being confused by this, maybe we should pick some different-enough name for `async_iterable`?

Reply:
You can try the change and see the linter complain, but basically explained here https://github.com/whatwg/webidl/issues/1490

Reply:
I opened #1504 to talk about this. I think it's *incredibly* confusing to have these two identical concepts spelled differently solely due to a grammar technicality that happens to only apply to one of them.